test/library/client.go

```diff
@@ -22,3 +25,14 @@ func NewClientConfigForTest(t *testing.T) *rest.Config {
 	require.NoError(t, err)
 	return config
 }
+
+// GenerateOAuthTokenPair returns two tokens to use with OpenShift OAuth-based authentication.
+// The first token is a private token meant to be used as a Bearer token to send
+// queries to the API, the second token is a hashed token meant to be stored in
+// the database.
+func GenerateOAuthTokenPair() (privToken, pubToken string) {
+	const sha256Prefix = "sha256~"
+	randomToken := base64.RawURLEncoding.EncodeToString([]byte(fmt.Sprintf("nottoorandom%d", mathrand.Int())))
```

Comment:
ok, bearer tokens should be base64 encoded.

Reply:
Funny you noticed, I think this is actually copy-paste where only certain parts were changed in order to make this simpler for tests. I'll make this simpler not to waste that precious CPU time